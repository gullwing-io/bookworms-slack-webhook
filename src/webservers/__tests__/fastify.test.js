// these tests are not working but the server is on local so for now its ok

// import { jest } from "@jest/globals";
// import request from "supertest";
// import app from "../fastify";

// // need to build server

// describe("Fastify server test", () => {
//   test("should return bookworms response from post", async () => {
//     await app.ready();

//     const response = await request(app.server)
//       .post("/webhooks/slack/bookworms")
//       .send({ text: "all" })
//       .set("Accept", "application/json");
//     expect(response.status).toEqual(200);
//     expect(response.text).toEqual();

//     app.close();
//   });
// });
