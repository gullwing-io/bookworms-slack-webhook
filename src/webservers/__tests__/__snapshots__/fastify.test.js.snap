// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fastify server test should return bookworms response from post 1`] = `"{\\"blocks\\":[{\\"type\\":\\"header\\",\\"text\\":{\\"type\\":\\"plain_text\\",\\"text\\":\\"Bookmarks\\"}},{\\"type\\":\\"divider\\"},{\\"type\\":\\"section\\",\\"text\\":{\\"type\\":\\"mrkdwn\\",\\"text\\":\\"Bookmarks are seperated into different domains, to get the specific bookmarks you can enter the following commands\\\\n\\\\n\`/bookmarks folder 1\`\\\\r\\\\n\`/bookmarks folder 2\`\\\\n\\"}}]}"`;
